source/mainnet/net/installation/downloads.rst

```diff
@@ -1 +1,118 @@
-.. include:: /shared/net/installation/downloads.rst
+
+.. _downloads:
```

Comment:
This label should be moved to above the first header in this file. 
Otherwise `variables.rst` could potentially change link of the label